modules/hosted-control-planes-overview.adoc

```diff
@@ -17,11 +17,11 @@ include::snippets/technology-preview.adoc[]
 [id="hosted-control-planes-architecture_{context}"]
 == Architecture of hosted control planes
 
-Typically, {product-title} is deployed in a coupled model, where a cluster consists of a control plane and a data plane. The control plane includes an API endpoint, a storage endpoint, a workload scheduler, and an actuator that ensures state. The data plane includes compute, storage, and networking where workloads and applications run.
+Typically, {product-title} is deployed in a coupled, or standalone model, where a cluster consists of a control plane and a data plane. The control plane includes an API endpoint, a storage endpoint, a workload scheduler, and an actuator that ensures state. The data plane includes compute, storage, and networking where workloads and applications run.
```

Comment:
s/or standalone/or standalone, ?

Reply:
Good catch! There should be a comma after "standalone". I'll add that.